src/govuk/helpers/_font-faces.scss

```diff
@@ -19,25 +17,25 @@
   @include govuk-not-ie8 { // In IE8, which cannot render WOFF format, we fall back to system fonts
     @include govuk-exports("govuk/helpers/font-faces") {
       @at-root {
-        /*! Copyright (c) 2011 by Margaret Calvert & Henrik Kubel. All rights reserved. The font has been customised for exclusive use on gov.uk. This cut is not commercially available. */
+        //! Copyright (c) 2011 by Margaret Calvert & Henrik Kubel. All rights reserved. The font has been customised for exclusive use on gov.uk. This cut is not commercially available.
```

Comment:
Just to double check – this copyright notice still appears in the compiled CSS?

Reply:
Yep 👍 I ran `npm run build:package` and the file still contains the notice:

<img width="697" alt="Screenshot 2020-08-25 at 11 57 34" src="https://user-images.githubusercontent.com/29889908/91166379-31b5c600-e6ca-11ea-8b1c-3cfad1723a1b.png">

Reply:
And in the compiled CSS in `dist/` as well?

Reply:
Good catch, it wasn't being included there. Looks like you can only include comments with `/*!` so I've reverted that line and included `stylelint-disable scss/comment-no-loud`

Reply:
:+1:

I'm wondering if that's worth writing a test for in `after-build-dist.test.js` – I think it's pretty important that the copyright notice stays intact. Do you think it'd make sense to do that as part of this PR?